src/platform/suecreek/include/platform/memory.h

```diff
@@ -277,16 +277,10 @@
 	SRAM_BANK_SIZE)
 #define SOF_CORE_S_T_SIZE ((PLATFORM_CORE_COUNT - 1) * SOF_CORE_S_SIZE)
 
-#define HEAP_BUFFER_SIZE	0xF000
+#define HEAP_BUFFER_SIZE	0x10000
```

Comment:
Why such value?

Reply:
It is full bank size - aligment is easier.

Reply:
@dabekjakub Thanks for info